packages/hash/frontend/src/components/layout/PageHeader/SearchBar/index.tsx

```diff
@@ -46,16 +50,35 @@ const toBlockUrl = (searchPage: PageSearchResult): string => {
   return segments.join("");
 };
 
-const ResultList: React.FC = (props) => (
-  <ul
-    className={tw`absolute z-10 w-1/2 max-h-60 overflow-auto border border-gray-100 rounded-lg shadow-md`}
+const ResultList: React.FC<{
+  isMobile: boolean;
+}> = ({ isMobile, ...props }) => (
+  <Box
+    component="ul"
+    sx={(theme) => ({
+      position: !isMobile ? "absolute" : "unset",
+      zIndex: 10,
+      width: "100%",
+      maxHeight: "15rem",
+      overflow: "auto",
+      border: `1px solid ${theme.palette.gray[20]}`,
+      borderRadius: "",
+    })}
+    className={tw`z-10 w-1/2 max-h-60 overflow-auto border border-gray-100 rounded-lg shadow-md`}
     {...props}
   />
 );
 
-const ResultItem: React.FC = (props) => (
-  <li
-    className={tw`flex border border-gray-100 bg-gray-50 p-2 hover:bg-gray-100 cursor-pointer overflow-ellipsis overflow-hidden`}
+const ResultItem: React.FC<{
+  sx?: SxProps<Theme>;
+}> = ({ sx, ...props }) => (
+  <Box
+    component="li"
+    sx={{
+      display: "flex",
+      ...sx,
+    }}
+    className={tw`border border-gray-100 bg-gray-50 p-2 hover:bg-gray-100 cursor-pointer overflow-ellipsis overflow-hidden`}
```

Comment:
Can we get rid of the tailwind styling whilst we're already here?

Reply:
Done here: https://github.com/hashintel/hash/commit/e73d00d0e7cc7935db76895e860580f755b53a2c